api/cpp/cbindgen.rs

```diff
@@ -391,6 +392,7 @@ fn gen_corelib(
         "Coord",
         "LogicalRect",
         "LogicalPoint",
+        "LogicalPosition",
```

Comment:
I'm surprised this hasn't show up earlier :)

Reply:
I had to add some code so LogicalPosition can be used in a `Property<>` in our items. This was not the case before this PR (see also the change in rtti and interpreter's Value)